script/foundry/utils/DeployHelper.sol

```diff
@@ -500,13 +500,11 @@ contract DeployHelper is Script, BroadcastManager, JsonDeploymentHandler, Storag
         _postdeploy("IpRoyaltyVaultBeacon", address(ipRoyaltyVaultBeacon));
 
         _predeploy("CoreMetadataModule");
+        impl = address(new CoreMetadataModule(address(accessController), address(ipAssetRegistry)));
         coreMetadataModule = CoreMetadataModule(
-            create3Deployer.deploy(
-                _getSalt(type(CoreMetadataModule).name),
-                abi.encodePacked(
-                    type(CoreMetadataModule).creationCode,
-                    abi.encode(address(accessController), address(ipAssetRegistry))
-                )
+            TestProxyHelper.deployUUPSProxy(
+                impl,
+                abi.encodeCall(CoreMetadataModule.initialize, address(protocolAccessManager))
```

Comment:
Need to pass `create3Deployer` to `TestProxyHelper.deployUUPSproxy()` in order to deploy proxy with CREATE3. Please refer to IPAssetRegistry deployment script between LN221 - LN238
```solidity
        ipAssetRegistry = IPAssetRegistry(
            TestProxyHelper.deployUUPSProxy(
                create3Deployer,
                _getSalt(type(IPAssetRegistry).name),
                impl,
                abi.encodeCall(IPAssetRegistry.initialize, address(protocolAccessManager))
            )
        );
```

Reply:
As all contracts are deployed with create3 now. the `CoreMetadataModule` also need to be deployed with new `[deployUUPSProxy](https://github.com/storyprotocol/protocol-core-v1/blob/e339f0671c9172a6699537285e32aa45d4c1b57b/test/foundry/utils/TestProxyHelper.sol#L16)()` which support create3. 

You might refer to IPAssetRegistry deployment code:
https://github.com/storyprotocol/protocol-core-v1/blob/e339f0671c9172a6699537285e32aa45d4c1b57b/script/foundry/utils/DeployHelper.sol#L221

Reply:
a692a589464505ecc70f7508c4fd042eb15b8a27 is the correct commit